pennylane/vqe/vqe.py

```diff
@@ -211,7 +211,4 @@ def __init__(self, ansatz, hamiltonian, device, interface="autograd", diff_metho
         self.cost_fn = qml.dot(coeffs, self.qnodes)
 
     def __call__(self, *args, **kwargs):
-        if self.qnodes.interface == "autograd":
-            return self.cost_fn(*args, **kwargs)[0]
-
```

Comment:
(`np.stack` is smarter than `np.vstack`, and doesn't create a vector when stacking a single element)